bin/init-ext.sh

```diff
@@ -4,6 +4,14 @@
 #
 set -ex
 
+install_requirements_file () {
+    if [ "$TOOL" = "uv" ]; then
+        uv pip install --system -r "$1"
+    else
+        pip install -r "$1"
```

Comment:
do we also need
--upgrade-strategy only-if-needed

Reply:
Nah. That was an attempt to deal with a dependency conflict issue around CKAN 2.9, but it didn't work and I eventually gave up and dropped 2.9.